docs/components/concepts/clusters.md

```diff
@@ -0,0 +1,44 @@
+---
+id: clusters
+title: "Clusters"
+description: "Learn more about the clusters available in your Camunda Platform 8 plan."
+---
+
+A cluster is a provided group of nodes that run Camunda Platform 8. By default, Camunda Platform 8 clusters are production-ready.
+
+Professional and Enterprise plan customers can create as many production or development clusters as they want.
+
+The following table shows each plan and available type or size of cluster:
+
+|              | Development | Production - S | Production - M | Production - L |
+| ------------ | ----------- | -------------- | -------------- | -------------- |
+| Free         | \-          | \-             | \-             | \-             |
+| Free Trial   | \-          | X              | \-             | \-             |
+| Professional | X           | X              | \-             | \-             |
+| Enterprise   | X           | X              | X              | X              |
+
+## Production clusters
+
+Production clusters come in three sizes: small (S), medium (M), and large (L). To learn more about the size of cluster best suited for your use case, see our [Best Practices](/components/best-practices/best-practices-overview.md) for more information on [sizing your runtime environment](/components/best-practices/architecture/sizing-your-environment.md#sizing-your-runtime-environment).
+
+When you deploy and execute your [BPMN](/components/modeler/bpmn/bpmn.md) or [DMN](/components/modeler/dmn/dmn.md) models on a production cluster, this might impact your monthly (Professional) or annual (Enterprise) total fee, meaning the more you execute your models, the higher your total fee may be.
+
+### Free Trial cluster
+
+A Free Trial cluster is a special kind of production cluster. Free Trial clusters are size small, and only available during your trial period.
```

Comment:
"A Free Trial cluster is a special kind of production cluster." -- This is a bit vague. What exactly do we mean by this? Unless we want to specify further, I would likely remove this sentence.

Reply:
We can remove this sentence or combine it with the next sentence. Which do you prefer?

Somewhat rephrasing but extra context - Free Trial clusters have all the features/functionality of a production cluster, but are size small and only available during your trial period. You cannot convert a Free Trial cluster to a different kind of cluster.

Reply:
Adding in commit where I'll also backport for 8.1